packages/karma-typescript/src/karma/reporter.ts

```diff
@@ -81,7 +84,15 @@ export class Reporter {
                             // @ts-ignore
                             .execute(context);
                     });
-                });
+
+                    const isCoverageCheckFailed = results && config.hasCoverageThreshold && !threshold.check(browser, remappedCoverageMap);
+
+                    isBelowCoverageThreshold = isBelowCoverageThreshold || isCoverageCheckFailed;
```

Comment:
@erikbarke `isCoverageCheckFailed` is the local `browser` instance check, where as `isBelowCoverageThreshold` is the aggregate for entire run.

Now to expand a bit on why I aggregate rather than do a hard exit: I do this in order to allow all of the coverage reports to be logged before the reporter triggers a process exit 🤓

Reply:
Ok, got it!